### ISSUE TITLE
Default Keyboard Layout

### ISSUE THREAD

**Issue by sodface** [TAAPArthur/navboard#5]
The README states that the qwerty layout is the default:

https://github.com/TAAPArthur/navboard/blob/6eb8e12a02789901ca9fb4f296f00e7d81061966/README.md?plain=1#L25

When I run navboard without any arguments, I get the mobile_symb layout, not qwerty. The mobile_symb layout is also first in the list which doesn't seem like a coincidence:

```
sodco:~$ navboard -l
mobile_symb
mobile
number
qwerty
alarm
clock
colors
sliders
spawn
board_of_boards
```

Is this where it's happening?
https://github.com/TAAPArthur/navboard/blob/5b5ac7f8adbd04ca5d08c1856870c78f47183a9c/navboard.c#L413

If I set `NAVBOARD_DEFAULT=qwerty` that works but I don't think that's what you meant in the README?

**Comment by TAAPArthur** [comment]
Seems like I put the wrong value in the readme. Although not really sure what should be the default. I'm tempted to make the board selection the default. Any thoughts?

**Comment by sodface** [comment]
Board selection as in board_of_boards? If so, that seems good to me.

**Comment by TAAPArthur** [comment]
Think this should be fixed by efb8023

**Comment by sodface** [comment]
Tested on x86_64, looks good to me. Thanks!
